### PR TITLE
fix(DB/locales): Restore deleted quest locales

### PR DESCRIPTION
**Pull request by LukasVolgger**
## CHANGES PROPOSED:
Restore deleted quest locales deleted in this commit: https://github.com/azerothcore/azerothcore-wotlk/commit/3f0392554017b46e03848af560ec5d0d82b138ed caused by a faulty `DELETE` statement

## ISSUES ADDRESSED:
- None

## TESTS PERFORMED:
- Checked some texts before deleting and after deleting

## HOW TO TEST THE CHANGES:

No tests required, can be merged after approval

## KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1605908814314990400.sql [hunk 1]
```diff
@@ -0,0 +1,2469 @@
...
..., 13916, 13917, 13926, 13927, 13929, 13930, 13931, 13932, 13933, 13934, 13937, 13938, 13950, 13951, 13954, 13955, 13956, 13957, 13959, 13960, 13986, 14016, 14017, 14074, 14076, 14077, 14080, 14081, 14082, 14083, 14084, 14085, 14086, 14087, 14088, 14090, 14092, 14095, 14096, 14100, 14101, 14102, 14104, 14105, 14107, 14108, 14111, 14112, 14136, 14140, 14141, 14142, 14143, 14144, 14145, 14151, 14152, 14163, 14164, 14166, 14169, 14171, 14176, 14183, 14203, 14349, 14352, 14356, 14409, 14443, 14444, 14457, 20438, 20439, 24217, 24429, 24431, 24442, 24451, 24454, 24461, 24476, 24480, 24498, 24499, 24506, 24510, 24522, 24535, 24536, 24541, 24553, 24554, 24555, 24556, 24557, 24558, 24559, 24560, 24561, 24562, 24563, 24564, 24576, 24579, 24580, 24581, 24582, 24583, 24584, 24585, 24586, 24587, 24588, 24589, 24590, 24594, 24598, 24609, 24610, 24611, 24612, 24613, 24614, 24615, 24655, 24656, 24657, 24665, 24666, 24683, 24710, 24711, 24796, 24800, 24801, 24802, 24804, 24805, 24819, 24820, 24821, 24822, 24831, 24833, 24836, 24837, 24838, 24839, 24840, 24841, 24842, 24843, 24844, 24845, 24846, 24847, 24848, 24849, 24850, 24851, 25239, 25246, 25247, 25248, 25249, 26012, 26013) AND `locale` = 'frFR';
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1605908814314990400.sql:6]: space before where

File: data/sql/updates/pending_db_world/rev_1605908814314990400.sql [hunk 2]
```diff
@@ -0,0 +1,2469 @@
...
+(11250, 'frFR', 'That''s unbelievably good news! I knew that you could pull it off, &lt;name&gt;!$B$BAt least someone around here can produce results!$B$BWell, this calls for a reward. Go ahead, take your pick; the treasury is open to you.', 18019),
+(11251, 'frFR', 'Bon, z''êtes peut-être en retard, mais ça fait du bien de voir un visage ami.$b$bBon, maintenant, arrêtez de rester planté là, à bâiller aux corneilles… Y a du travail à faire !', 18019),
+(11252, 'frFR', 'Outstanding!$B$BThis could be the turn of luck we''ve all been hoping for.$B$BWell done, &lt;name&gt;!', 18019),
+(11255, 'frFR', 'Les soldats que vous avez sauvés avaient des informations intéressantes. Ces monstres s’appellent des « vrykuls ». La tribu qui s’est installée au village de Crâne-du-ver est dirigée par un mystique nommé Yanis, qui travaille pour un autre vrykul, un certain Ingvar le Pilleur.', 18019),
+(11260, 'frFR', 'Oui, c’est sinistre, et vous avez certainement eu raison.$b$bMerci de m’avoir remis ce parchemin, je vais l’étudier. Si je devais hasarder une hypothèse, je dirais que ces vrykuls ont un moyen de transcender la mort – ceux d’entre eux qui en sont jugés dignes, en tout
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1605908814314990400.sql:2469]:
```suggestion
INSERT INTO `quest_request_items_locale` (`ID`, `locale`, `CompletionText`, `VerifiedBuild`) VALUES 
(9483, 'frFR', 'Ce n\'est pas parce que je suis en pèlerinage que je dois abandonner tous les plaisirs de la vie. Cela fait des siècles que je n\'ai pas degusté un bon vin.$b$bJ\'en arriverais presque à tuer pour une bouteille d\'esprit-de-vin cénarien !$b$bJe sais, je sais, ce n\'est pas du tout la mode de boire quelque chose produit par les elfes de la nuit.', 18019);
```

Comment (LukasVolgger) [reply data/sql/updates/pending_db_world/rev_1605908814314990400.sql:2469]: Not sure if this is correct as you suggested, better keep the original like before

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by Kitzunu** [comment]
Thank you!

**Pull request merged**